### PR TITLE
chown: add example to French translation

### PR DESCRIPTION
**Pull request by Gabrinator**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by Nico385412 — APPROVED** [review]
LGTM

File: pages.fr/common/chown.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
 `chown {{utilisateur}}:{{groupe}} {{chemin/vers/fichier_ou_dossier}}`
 
+- Modifie le propriétaire et le groupe pour qu'ils aient les deux le nom `utilisateur` :
```
Comment (nicokosi) [inline pages.fr/common/chown.md:14]: I don't understand. :thinking:

PS; the English version is:
> Change the owner user and group to both have the name user:

Comment (nicokosi) [reply pages.fr/common/chown.md:14]: What about this?

> Modifie le propriétaire et le groupe pour qu'ils aient tous les deux le nom `utilisateur` :

or :

> Modifie le propriétaire et le groupe pour qu'ils aient le nom `utilisateur` :

Comment (Gabrinator) [reply pages.fr/common/chown.md]: I think your first suggestion is clearer. Updated! Thanks!

**Review by spageektti — APPROVED** [review]

**Review by nicokosi — APPROVED** [review]
:+1:

**Pull request merged**